tests/x11/ghostscript.pm

```diff
@@ -64,8 +62,14 @@ sub run {
     # check if there was an error during the pdf generation
     assert_script_run "test ! -f $gs_failed";
 
+    # exit root console for switching to x11
+    type_string "exit\n";
```

Comment:
`select_console()` or `select_serial_terminal()` close the console session automatically. `type_string "exit\n";` was there in the original code to exit from `su` session back into parent shell. You're not running a nested shell here.

Reply:
yes, updated now, thanks.